server/server.go

```diff
@@ -73,6 +75,13 @@ func instanceIdHandler(w http.ResponseWriter, r *http.Request) {
 	fmt.Fprintf(w, "aws-vault")
 }
 
+func regionDiscoveryHandler(w http.ResponseWriter, r *http.Request) {
+	//the config source profile has a region defined, this should be used here
+	json.NewEncoder(w).Encode(map[string]interface{}{
+		"region": "us-west-2",
```

Comment:
i can't figure out how to get the config imported to the server module that contains the region from the config file, obviously we can't merge this hard coded version...

Reply:
this code is a separate process to the actual credentials, so unless it gets proxied through as the credentials do its a bit stuck. Even if we pass its as a param on startup people may want to run different profiles with different regions and as this process is long lived thats not going to work.

urgh.

Reply:
I reckon checking the AWS_REGION and AWS_DEFAULT_REGION environment would be legit, or falling back to us-east-1.